test/io/jesi/backpack/fn_test.cljc

```diff
@@ -137,7 +137,7 @@
   (testing "and-fn"
 
     (testing "throws an exception if no parameters are given"
-      (is (thrown? #?(:clj ArityException :cljs js/Error) (bp/and-fn))))
+      (is (thrown? #?(:clj ArityException :cljs js/Error) (eval `(bp/and-fn)))))
```

Comment:
why eval?

Reply:
Lint (like eastwood) does not like calling the function with incorrect arity. This way we can still test the behaviour but without the linter crying.